apps/web-giddh/src/app/gst/gst.component.scss

```
@@ -517,12 +511,24 @@ hr {
     .return-content .content-graph {
         padding: 15px 0;
     }
+    .gstr-1-lable, .gstr-2-lable{
```

Do use correct spelling 
```suggestion
    .gstr-1-label, .gstr-2-label{
```

---

Done